# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/relations/browser_anchor_positioning.js
# Commit: 67f28439cbc7
# Full Hash: 67f28439cbc71117a690ea3eee3ef12b0593a7e6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-29 20:54 +0000
# Regressor Bug: 1987945
# File Overlap Count: 1
# Overlapping Files: accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P3: Support details relation in anchored targets in remote accessibles. r=Jamie,layout-anchor-positioning-reviewers,layout-reviewers,emilio,firefox-style-system-reviewers
# ==============================================================================

diff -r b93b02b24271 -r 67f28439cbc7 accessible/tests/browser/relations/browser_anchor_positioning.js
--- a/accessible/tests/browser/relations/browser_anchor_positioning.js	Mon Sep 29 20:54:29 2025 +0000
+++ b/accessible/tests/browser/relations/browser_anchor_positioning.js	Mon Sep 29 20:54:29 2025 +0000
@@ -127,7 +127,7 @@
 
     await testNoDetailsRelations(btn2, target2);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -162,7 +162,7 @@
 
     await testDetailsRelations(siblingBtn, siblingTarget);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -218,7 +218,7 @@
       await testDetailsRelations(ownerBtn, ownedTarget);
     }
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -357,7 +357,7 @@
     await invokeSetAttributeAndTick(browser, "multiAnchor-btn1", "hidden");
     await testNoDetailsRelations(multiAnchorBtn2, multiAnchorTarget);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -387,7 +387,7 @@
     const tooltipTarget = findAccessibleChildByID(docAcc, "tooltip-target");
     await testNoDetailsRelations(btn, tooltipTarget);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -509,7 +509,7 @@
     );
     await testDetailsRelations(btnTargetsetdetails, targetTargetsetdetails);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );
 
 /**
@@ -565,5 +565,5 @@
     );
     await testNoDetailsRelations(anchor1, target);
   },
-  { chrome: true, topLevel: false }
+  { chrome: true, topLevel: true }
 );